graphql_compiler/tests/test_helpers.py

```diff
@@ -275,6 +276,20 @@ def get_schema():
     """Get a schema object for testing."""
     ast = parse(SCHEMA_TEXT)
     schema = build_ast_schema(ast)
+    # The schema text contains no information about how to parse or serialize non-builtin scalar
+    # types so we add this information manually. For more info see the link below:
+    # pylint: disable=max-line-length, protected-access
+    # https://stackoverflow.com/questions/47824603/graphql-custom-scalar-definition-without-graphql-tools
```

Comment:
I think the link is probably not necessary, the description is sufficient here. Let's try to avoid the pylint suppressions if we can -- I think you can get around the protected access one by looking up type objects properly.